include/roboteam_ai/utilities/Dealer.h

```diff
@@ -50,6 +50,8 @@ class Dealer {
   using FlagMap = std::map<std::string, std::vector<DealerFlag>>;
   Dealer(v::WorldDataView world, world::Field * field);
   virtual ~Dealer() = default; // needed for test
+  // Create a distribution of robots
+  std::unordered_map<std::string, v::RobotView> distribute(const std::vector<v::RobotView> &allRobots, const FlagMap &flagMap);
```

Comment:
why std::string

Reply:
This is the role_name the robot is distributed to. We might be able to make some better solution for this but that is a refactor for a later PR if necessary